lindera/src/tokenizer.rs

```diff
@@ -94,6 +94,14 @@ impl TokenizerBuilder {
         })
     }
 
+     pub fn from_config(config: TokenizerConfig) -> LinderaResult<Self> {
+        println!("config: {:?}", config);
```

Comment:
Can you please remove this` println()` as well, since it is a leftover debugging code?